### PR TITLE
Debian fixes last

### PR DESCRIPTION
**Pull request by rleon**

### CODE REVIEW

File: debian/libibverbs1.symbols [hunk 1]
```diff
@@ -1,6 +1,8 @@
 libibverbs.so.1 libibverbs1 #MINVER#
  IBVERBS_1.0@IBVERBS_1.0 1.1.6
  IBVERBS_1.1@IBVERBS_1.1 1.1.6
+ IBVERBS_1.3@IBVERBS_1.3 1.1.12
```
Comment (jgunthorpe) [inline debian/libibverbs1.symbols]: These are all still wrong.

Comment (rleon) [inline debian/libibverbs1.symbols]: @talatb I double and triple checked it and this patch exactly as you forwarded it to me.

Comment (jgunthorpe) [inline debian/libibverbs1.symbols]: FWIW, #65 will eliminate the need for this patch...

File: debian/libibverbs1.symbols [hunk 2]
```diff
@@ -15,27 +17,33 @@ libibverbs.so.1 libibverbs1 #MINVER#
  ibv_cmd_alloc_pd@IBVERBS_1.0 1.1.6
  ibv_cmd_attach_mcast@IBVERBS_1.0 1.1.6
  ibv_cmd_close_xrcd@IBVERBS_1.1 1.1.8
- ibv_cmd_create_ah@IBVERBS_1.0 1.1.6
+ ibv_cmd_create_ah@IBVERBS_1.4 12
```
Comment (bdrung) [inline debian/libibverbs1.symbols]: This ABI breakage needs to be discussed and decided (see mailing list).

**Comment by rleon** [comment]
Let's drop this series completely and wait till ABI discussion will be closed.
@bdrung, @jgunthorpe and @talatb what do you think?

**Comment by bdrung** [comment]
Yes, I agree to wait for the ABI discussion to be closed before changing anything.

**Comment by jgunthorpe** [comment]
The init.d script patch seems fine...

**Comment by bdrung** [comment]
Right. Since both init scripts already depend on $network, adding a dependency to $remote_fs is safe.

**Comment by bdrung** [comment]
Result from my side:

* debian: Update libibverbs symbols file -> wait for ABI discussion results
* debian: Update to debhelper 10 -> wait for end-of-life of jessie
* debian: Fix init.d script missing dependency on remote_fs -> accept
* debian: Switch to automatically created -dbgsym packages. -> wait for end-of-life of jessie

**Comment by rleon** [comment]
@bdrung, I'll cherry-pick that init.d patch.
This pull request is closed.

Thanks

**Pull request closed without merging**